depinject/internal/appconfig/registry.go

```diff
@@ -35,12 +38,18 @@ func ModulesByModuleTypeName() (map[string]*ModuleInitializer, error) {
 		fullName := gogoproto.MessageName(initializer.ConfigProtoMessage)
 
 		if desc, err := gogoproto.HybridResolver.FindDescriptorByName(protoreflect.FullName(fullName)); err == nil {
-			modDesc := protov2.GetExtension(desc.Options(), appv1alpha1.E_Module).(*appv1alpha1.ModuleDescriptor)
+			dp := protodesc.ToDescriptorProto(desc.(protoreflect.MessageDescriptor))
```

Comment:
compile err here, unused var

Reply:
Sorry I will clean it up. Was debugging and forgot to delete